src/main/java/com/bp3/camunda/camunda7/C7RESTConnector.java

```diff
@@ -63,6 +68,19 @@ public void execute(ExternalTask externalTask, ExternalTaskService externalTaskS
         Map<String, String> httpHeaders = (Map<String, String>) getVariable(externalTask, PARAM_HTTP_HEADERS, Map.class);
         Map<String, String> httpQueryParams = (Map<String, String>) getVariable(externalTask, PARAM_HTTP_PARAMETERS, Map.class);
         String outputVariableName = (String) getVariable(externalTask, PARAM_OUTPUT_VARIABLE, String.class);
+        String errorHandlingMethod = (String) getVariable(externalTask, PARAM_ERROR_HANDLING_METHOD, String.class);
+
+        int totalRetries = 0; // Default to zero if not set
+        String retriesParam = (String) getVariable(externalTask, PARAM_RETRIES, String.class);
+        if (Objects.nonNull(retriesParam)) {
```

Comment:
Objects.nonNull() is intended for use in a filter predicate, just use simple boolean operators here
See https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#isNull-java.lang.Object-
```suggestion
        if (retriesParam != null) {
```

Reply:
Updated